### PR TITLE
add sudo to commands

### PR DESCRIPTION
**Pull request by BO41**
```
createdb -O peertube peertube_test1   
createdb: database creation failed: ERROR:  must be member of role "peertube"
```

### CODE REVIEW

**Comment by Chocobozzz** [comment]
Did you execute `sudo -u postgres createuser you_username --createdb --superuser`?

**Comment by BO41** [comment]
probably without `--superuser` as I accidentally created this user already.
Would this fix the error? Then sorry for this PR, please close :)

**Comment by Chocobozzz** [comment]
> Would this fix the error?

yes

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -140,7 +140,7 @@ Then, we can create the databases (if they don't already exist):
 
 ```
 $ sudo -u postgres createuser you_username --createdb
```
Comment (BO41) [inline .github/CONTRIBUTING.md]: @Chocobozzz at first I ran this command, thats the reason the user already existed. And here is no `--superuser`.
So theoretically, here `sudo` would be needed, right?

**Pull request closed without merging**